src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int TABLE_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

It will be initialized by 0 by default